docs/unpacked-format.rst

```diff
@@ -0,0 +1,120 @@
+
+This page describes in more details how the unpackers operate.
+
+..  note:: Future versions of unpackers might work in a different way. No attempt is made to make unpacked experiments compatible across different versions of *reprounzip*.
+
+..  _unpacked-directory:
+
+The `directory` Unpacker
+========================
+
+The experiment directory contains:
+
+* The original configuration file ``config.yml``.
+* The pickle file ``.reprounzip``.
+* The tarball ``inputs.tar.gz``, which contains the original files that were identifies as input files. This tarball is used for file restoration using ``upload :<input-id>`` (see :ref:`unpacker-input-output`).
+* A directory called ``root``, which contains all the packaged files in their original path, with symbolic links to absolute paths rewritten to prepend the path to ``root``.
+
+::
+
+    unpacked-directory/
+        .reprounzip
+        config.yml
+        inputs.tar.gz
+        root/
+            ...
+
+When running the ``run`` command, the unpacker sets ``LD_LIBRARY_PATH`` and ``PATH`` to point inside ``root``, and optionally ``DISPLAY`` and ``XAUTHORITY`` to the host's ones.
+
+..  _unpacked-chroot:
+
+The `chroot` Unpacker
+=====================
+
+The experiment directory contains:
+
+* The original configuration file ``config.yml``.
+* The pickle file ``.reprounzip``, which stores whether magic directories are mounted, as explained below.
+* The tarball ``inputs.tar.gz``, which contains the original files that were identifies as input files. This tarball is used for file restoration using ``upload :<input-id>`` (see :ref:`unpacker-input-output`).
+* A directory called ``root``, which contains all the packaged files in their original path, with no symbolic links rewritten and file ownership restored.
+
+::
+
+    unpacked-directory/
+        .reprounzip
+        config.yml
+        inputs.tar.gz
+        root/
+            dev/
+            dev/pts/
+            proc/
+            ...
+
+If a file is listed in the configuration file but wasn't packed (i.e.: ``pack_files`` was set to ``false`` for a software package), such file is copied from the host; if this file does not exist on the host, a warning is shown when unpacking.
+
+Unless ``--dont-bind-magic-dirs`` is specified when unpacking, the special directories ``/dev``, ``/dev/pts``, and ``/proc`` are mounted with ``mount -o bind`` from the host.
+Also, if ``/bin/sh`` or ``/usr/bin/env`` weren't both packed, a static build of `busybox <https://busybox.net/>`__ is downloaded and put under ``/bin/busybox``, and the missing binaries are created as symbolic links pointing to busybox.
+
+Should you require a shell inside the experiment environment, you can use::
+
+    chroot root/ /bin/sh
+
+..  _unpacked-vagrant:
+
+The `vagrant` Unpacker
```

Comment:
@remram44: any reason for not showing the experiment structure as you did in the directory and chroot unpackers? Same for the docker unpacker.